packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md

```diff
@@ -223,7 +240,12 @@ export class LoginComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.widget.showSignInAndRedirect().catch(err => {
+    const originalUri = this.authService.getOriginalUri();
+    if (!originalUri) {
+      this.authService.setOriginalUri('/');
+    }
+    
+    this.widget.showSignInAndRedirect().catch((err: Error) => {
```

Comment:
I feel we should use `showSIgnInAndGetTokens` to get SPA experience. What this guide for?